--- libdigidoc-3.10.4/cmake/modules/VersionInfo.cmake~	2018-06-12 12:22:01.000000000 +0200
+++ libdigidoc-3.10.4/cmake/modules/VersionInfo.cmake	2018-09-14 20:59:46.952522826 +0200
@@ -81,7 +81,7 @@ if(NOT DEFINED ENABLE_VISIBILITY)
 	endif()
 endif()
 
-if(NOT DISABLE_CXX11)
+if(FALSE)
 	if(CMAKE_VERSION VERSION_GREATER  3.1.0)
 		set(CMAKE_CXX_STANDARD 11)
 		set(CMAKE_CXX_STANDARD_REQUIRED YES)
